scripts/auto-build

```diff
@@ -162,9 +162,9 @@ if ! scripts/make-with-log \
         DISTS_VM="$built_for_vm" \
         DIST_DOM0="$built_for_dom0" \
         BUILD_LOGS_URL="$build_logs" \
-        GIT_URL_$(echo $COMPONENT | tr - _)=$git_url \
+        GIT_URL_${COMPONENT//-/_}="$git_url" \
```

Comment:
Oops, we're in /bin/sh and ${...//.../...} [is not portable](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02). Reverting to leave as `$( ...| tr ...)`